### ISSUE TITLE
Staging Review finding: Include guidance on privacy masking for sensitive prefilled content.

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#95785]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Design System Staging Information
**Component:** Help users to... Know when their information is prefilled
**Staging Review ticket:** [AEDP, Design Patterns, Help users to... Know when their information is prefilled](https://github.com/department-of-veterans-affairs/va.gov-team/issues/90181)

## Findings details
**VA.gov Experience Standard - issue:** [User doesn't have enough information to complete a task.](https://design.va.gov/about/experience-standards/#user-has-enough-information-to-complete-a-task)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**This is an issue with the:** guidance
**High-priority:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

In the [Personal information page](https://design.va.gov/patterns/help-users-to/know-when-their-information-is-prefilled#personal-information-page) anatomy section, the Veteran's Social Security number is displayed with the first five digits masked. I don't think we have anywhere in the design system that currently tells VFS teams how to mask digits like this in an accessible way, and this feels to me like an appropriate place to do it.

## Recommended action
Up to you to decide where this should be inserted --- I'm thinking possibly in the still pending Code Usage section or in a standalone Accessibility Considerations section ([example of an Accessibility Considerations section](https://design.va.gov/patterns/ask-users-for/a-single-response#accessibility-considerations)), but you obviously know where it should go.

When masking characters, what we definitely don't want is screen reader software reading "Social Security Number: star star star dash star star dash 1234." The visible text with masked values should be wrapped in a `<span aria-hidden="true">` (this tells screen readers to ignore it), and then followed immediately by something like `<span class="sr-only">Number ending in 1234</span>` (this tells screen readers what to announce instead).

I would also encourage you to work with CAIA to establish guidance on what kinds of data should/shouldn't be masked for privacy, and include that here as well. But at a minimum I think the code guidance would be a huge benefit.

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for DST

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Governance help validating the issue, please comment on the ticket
- If this ticket has a `high-priority` label, please address as soon as possible so VFS teams are not impacted
- If this ticket has a `consider` label, please consider for future implementation

**Comment by msbtterswrth** [comment]
We are tracking this update in [this task](https://github.com/department-of-veterans-affairs/tmf-auth-exp-design-patterns/issues/165) in our team repo.

**Comment by msbtterswrth** [comment]
I chatted with Matt Dingee about this a bit, and we think that putting this guidance in the prefill pattern page doesn't make sense. We'll instead make a recommendation and open a PR to update the existing [Social Security number pattern](https://design.va.gov/patterns/ask-users-for/social-security-number).

**Comment by shiragoodman** [comment]
@briandeconinck FYI
